lib/cosmos/dart/lib/dart_packet_log_writer.rb

```diff
@@ -55,6 +55,11 @@ def graceful_kill
 
   # Override the default new file hook to create a PacketLog entry in the database
   def start_new_file_hook(packet)
+    # When we create a new file we mark any existing PLEs ready
+    PacketLogEntry.where("id" => @not_ready_ple_ids).update_all(ready: true)
+    @not_ready_ple_ids.clear
+    @sync_count = 0
```

Comment:
I added this due to unit testing I was doing. You would eventually catch up and mark these ready but in the meantime you'd have a finished packet log file with entries marked not ready. Not an issue in DART due to the 2GB log size I guess. Thoughts?